source/dub/recipe/io.d

```diff
@@ -132,6 +140,23 @@ unittest { // issue #711 - configuration default target type not correct for SDL
 	}
 }
 
+unittest { // make sure targetType of sub packages are sanitized too
+	import dub.compilers.buildsettings : TargetType;
+	auto inputs = [
+		"dub.sdl": "name \"test\"\nsubPackage {\nname \"sub\"\ntargetType \"sourceLibrary\"\nconfiguration \"a\" {\n}\n}",
+		"dub.json": "{\"name\": \"test\", \"subPackages\": [ { \"name\": \"sub\", \"targetType\": \"sourceLibrary\", \"configurations\": [{\"name\": \"a\"}] } ] }"
```

Comment:
You might want to use backtip-delimited strings here ?

Reply:
I just copy&pasted it from the other unittests above. Backticks would be usable for the JSON, yes, but not for the sdl (in a single line), so for consistency...